text_classifier/sigopt_creds.py

```diff
@@ -0,0 +1,4 @@
+# Find your creds at https://sigopt.com/user/profile
```

Comment:
Would we ever want to automatically set up this file for users? They could run `./setup_env.sh --client_token=CLIENT_TOKEN` and the rest is handled